### ISSUE TITLE
Fl_Output value changed by drag and drop

### ISSUE THREAD

**Issue by stgatilov** [fltk/fltk#113]
Hello again!

I have noticed a weird thing, which is probably a bug. As far as I understood, default Fl_Output is like text input field, but without allowing the user to change the text. User can scroll, select, and copy/paste the text, but not change it.

However, if I select the text and drag-and-drop it to same Fl_Output, then the selected substring disappears.
Here is [the video](https://www.youtube.com/watch?v=303jQwOev60).

It turns out that I should not keep valuable information in Fl_Output widgets =)

I'm using FLTK 1.3.5 on Windows 10, in case it matters.

**Comment by Albrecht-S** [comment]
Thanks for the report and for mentioning the FLTK version 1.3.5.

This has been fixed in FLTK 1.4.x (not yet released) in commit 3e4baae6fcb40c3a377c170af2f7698e18dc03e4.

I'm backporting it to FLTK 1.3.x (Git branch-1.3). I believe that we will release 1.3.6 soon, but this is not yet decided.

**Comment by Albrecht-S** [comment]
Fixed in Git commit d4859abe669d.

Please confirm that this fixes the issue for you (you will need to pull from Git or copy the [changed file](https://raw.githubusercontent.com/fltk/fltk/branch-1.3/src/Fl_Input.cxx) to your own FLTK sources).

**Comment by stgatilov** [comment]
I tried to rebuild FLTK 1.3.5 with the patch https://github.com/fltk/fltk/commit/d4859abe669db503303685dbcd30782d826c8f8b.
Yes, it fixes the problem.

Rebuilding was not trivial, since I have wrapped FLTK into [conan package](https://github.com/stgatilov/conan-fltk). I think I will add the patch to recipe in TheDarkMod, but won't add it to the linked GitHub repo. It's better to wait for 1.3.6 than offering people to install some mixed version.
